### DISCUSSION TITLE
position of  "touch points" of a 3D object

### DISCUSSION THREAD

**Discussion by inst5** [cables-gl/cables_docs#425] [Answers & Questions] [answered]
Hi,
I am trying to learn more about Ammo-Objects.
I want to understand and clone this patch SDF Based 3D Painting (Ammo) https://cables.gl/p/rMDPC6 to experiment with it.
but there is mentioned: "THIS PATCH HAS PRIVATE OPS. You can continue saving this patch, but probably some things will not work."

I am looking for examples  to get the position (XY) of the "touch points" when a 3D object is touched.
Kind like in the example above a continious data stream of the positions.

thanks for any feedback.

**Comment by pandrr** [comment, marked as answer]
the ops in that patch are made by a user. ask the user if he helps you with your endeavour.

no idea what he changed

**Comment by pandrr** [comment]
https://dev.cables.gl/user/amajesticseaflapflap

**Reply by inst5** [reply]
thanks for your feedback!
